test/FunctionalTests/Client/StreamingTests.cs

```diff
@@ -35,6 +36,17 @@ namespace Grpc.AspNetCore.FunctionalTests.Client
     [TestFixture]
     public class StreamingTests : FunctionalTestBase
     {
+        private const int BigMessageSize = 1024 * 1024 * 5;
```

Comment:
Where does this number come from?

Reply:
Its a message big enough to hit flow control if the peer doesn't immediately read it.

Added comment.